server/sonar-web/src/main/js/app/types.ts

```diff
@@ -166,19 +166,42 @@ export interface Group {
 }
 
 export enum HomePageType {
-  Project = 'PROJECT',
-  Organization = 'ORGANIZATION',
+  Application = 'APPLICATION',
+  Issues = 'ISSUES',
+  MyIssues = 'MY_ISSUES',
   MyProjects = 'MY_PROJECTS',
-  MyIssues = 'MY_ISSUES'
+  Organization = 'ORGANIZATION',
+  Portfolio = 'PORTFOLIO',
+  Portfolios = 'PORTFOLIOS',
+  Project = 'PROJECT',
+  Projects = 'PROJECTS'
 }
 
-export interface HomePage {
-  parameter?: string;
-  type: HomePageType;
-}
+// export interface HomePage {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this commented out code. ([typescript:S125](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS125))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fhomepage&issues=AWGU57fWQeDOAQ7I7r8c&open=AWGU57fWQeDOAQ7I7r8c)

[ISSUE_KEY]: AWGU57fWQeDOAQ7I7r8c